src/normal_path.rs

```diff
@@ -183,3 +183,22 @@ impl ValueParserFactory for NormalPath {
         Self::Parser::default()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::env;
+
+    use super::*;
+
+    #[test]
+    fn test_into_absolute() {
+        let dir = env::temp_dir();
```

Comment:
I suspect `env::temp_dir()` is already absolute, so I'm not sure how useful this is as a test. Looks good though.